docs/design/vehicle_dynamics/analysis.md

```diff
@@ -0,0 +1,71 @@
+# Vehicle Physics in O3DE - analysis
+
+## Types of robot mobility and the scope of analysis
+
+This analysis only includes **wheeled robots**.
+
+- Airborne robots 
+- Ground robots
+  - **Wheeled robots**
+    - **...**
+  - Walking robots 
+    - Bipedal (e.g., humanoid) 
+    - Quadrupedal (e.g., dogs)
+    - …
+  - Crawling robots (snake-like)
+  - Climbing robots  
+  - ... 
+- (Under)water robots 
+- Space robots
+- ...
+
+## Types of wheels
+
+There are many types of wheels and their mounting used in robotics. Some examples include:
+
+- **Fixed wheels** – fixed to the chassis. They are used to drive the robot and to steer it (using differential drive).
+- **Omni-wheels** (including Mecanum wheels): they are equipped with rollers which enable multi-directional movement. They are used both for driving and steering.
+- **Castor wheels** (and balls) - passive wheels used for balance. 
+- **Car-like wheels** – connected through axles, with additional steering elements, with tires. 
+
+## Selected robot steering types
```

Comment:
I would love to see more explanations of how the different types are implemented (o3de joints and rigid bodies or PhysX vehicle framework?).